### ISSUE TITLE
protoc-gen-go should not generate deprecated imports

### ISSUE THREAD

**Issue by TimSatke** [golang/protobuf#1119]
**What version of protobuf and what language are you using?**
```
// 	protoc-gen-go v1.22.0
// 	protoc        v3.11.4
```

**What did you do?**
`//go:generate protoc --go_out=. append_entries.proto`

**What did you expect to see?**
I expected to see a non-deprecated import statement, as the other import statements in that file are not deprecated packages, only the `proto` one
```
import (
    proto "google.golang.org/protobuf/proto"
    protoreflect "google.golang.org/protobuf/reflect/protoreflect"
    protoimpl "google.golang.org/protobuf/runtime/protoimpl"
    reflect "reflect"
    sync "sync"
```

**What did you see instead?**
A deprecated package import, which seems to work fine, but analysis tools generate warnings, which makes the commit fail in our CI.
```
import (
    proto "github.com/golang/protobuf/proto"
    ...
```

**Anything else we should know about your project / environment?**
The failing analysis tool is `staticcheck`

Also, I can accept if this is not a bug, however, I would like to know if there's a workaround to get a non-deprecated, still working import statement besides from editing it manually after generating.

**Comment by dsnet** [comment]
Duplicate of #1077. We need to keep this dependency at least some time to support users that are not yet using Go modules. We'll remove it a few months.

Also, I believe `staticcheck` has been modified to specially ignore this case.

**Comment by TimSatke** [comment]
Go modules were released with go 1.11, current version is 1.14.
Will you change that with the release of go 1.15 or is this independent?
